### PR TITLE
Make recipes mx8 COMPAT and remove unneeded files

### PR DESCRIPTION
**Pull request by chrisdimich**

### CODE REVIEW

File: recipes-libraries/tensorflow-lite/tensorflow-lite_2.3.1.bb [hunk 1]
```diff
@@ -108,3 +108,5 @@ INSANE_SKIP_${PN} += " \
 "
 
 FILES_${PN} += "${libdir}/python*"
+
+COMPATIBLE_MACHINE = "(mx8)"
```
Comment (otavio) [inline recipes-libraries/tensorflow-lite/tensorflow-lite_2.3.1.bb:112]: This commit is lacking Signed-off-by ...

**Comment by chrisdimich** [comment]
Apologies. Fixed now.

**Comment by otavio** [comment]
There are two more, also lacking it.

**Comment by chrisdimich** [comment]
Fixed.

**Review by otavio — APPROVED** [review]

**Pull request merged**